runway/blueprints/staticsite/auth_at_edge.py

```diff
@@ -92,47 +92,67 @@ def create_template(self):
         lambda_function_associations = []
 
         if self.directory_index_specified:
-            rewrite_role = self.add_index_rewrite_role()
-            index_rewrite = self.add_cloudfront_directory_index_rewrite(rewrite_role)
-            index_rewrite_version = self.add_cloudfront_directory_index_rewrite_version(
-                index_rewrite
-            )
+            index_rewrite = self._get_index_rewrite_role_function_and_version()
             lambda_function_associations = self.get_directory_index_lambda_association(
                 lambda_function_associations,
-                index_rewrite_version
+                index_rewrite['version']
             )
 
         # Auth@Edge Lambdas
-        lambda_execution_role = self.add_lambda_execution_role()
         check_auth_lambda = self.get_auth_at_edge_lambda_and_ver(
             'CheckAuth',
             'Check Authorization information for request',
             'check_auth',
-            lambda_execution_role
+            self.add_lambda_execution_role(
+                'CheckAuthLambdaExecutionRole',
+                Join(':', ['arn:aws:logs:*',
```

Comment:
Another non-functional change here (only relevant if someone copies the code to one of the isolated regions) -- should swap `aws` here for [Partition](https://github.com/cloudtools/troposphere/blob/2.6.0/troposphere/__init__.py#L515)